docs/developer/extending/plugins/openid_connect.mdx

```diff
@@ -0,0 +1,261 @@
+| -------------------------- | --------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| saleor:manage_staff        | MANAGE_STAFF                | Access to staff users data                                                                                                       |
+| saleor:manage_apps         | MANAGE_APPS                 | Manage apps                                                                                                                      |
+| saleor:manage_users        | MANAGE_USERS                | Access to customers data                                                                                                         |
+| saleor:manage_discounts    | MANAGE_DISCOUNTS            | Manage discounts                                                                                                                 |
+| saleor:manage_plugins      | MANAGE_PLUGINS              | Manage plugins                                                                                                                   |
+| saleor:manage_gift_card    | MANAGE_GIFT_CARD            | Manage gift cards                                                                                                                |
+| saleor:manage_menus        | MANAGE_MENUS                | Manage the structure of menus                                                                                                    |
+| saleor:manage_orders       | MANAGE_ORDERS               | Access to orders data                                                                                                            |
+| saleor:manage_pages        | MANAGE_PAGES                | Manage pages                                                                                                                     |
+| saleor:manage_product      | MANAGE_PRODUCTS             | Manage products                                                                                                                  |
+| saleor:manage_shipping     | MANAGE_SHIPPING             | Manage shipping                                                                                                                  |
+| saleor:manage_settings     | MANAGE_SETTINGS             | Manage shop settings                                                                                                             |
+| saleor:manage_translations | MANAGE_TRANSLATIONS         | Manage tranlsations                                                                                                              |
+| saleor:manage_checkouts    | MANAGE_CHECKOUTS            | Manage checkout                                                                                                                  |
+
+
+If the user has granted given permission on the OAuth side, Saleor will append the granted 
```

Comment:
Too much information, this description focuses on how it works internally instead of the observable effects.

"If permission delegation (not sure what it's called in the UI) is enabled, any permissions granted to a user on the authentication provider side will be used as the effective permissions on the Saleor side. Please see the documentation of your authentication server to see how to manage permissions and how to configure role-based access control (RBAC)."